exp/templates/exp/dashboard.html

```diff
@@ -5,16 +5,22 @@
 {% load bootstrap3 %}
 
 {% block title %}Dashboard{% endblock %}
-{% block header %}{% endblock %}
+{% block header %}
+  {% include 'exp/_navigation.html' %}
+{% endblock %}
 
 {% block content %}
 <div class="container">
+    {% bootstrap_messages %}
   <div class="panel panel-default">
     <div class="panel-heading">
       <h1>{% trans "Dashboard" %}</h1>
     </div>
     <div class="panel-body">
-      <p>You've successfully signed into Experimenter. An administrator will be contacted to grant you permissions. You will receive a notification by email when this is completed.</p>
+      <p>You've successfully signed into Experimenter! </p>
+      <p>
+          An administrator will be contacted to grant you permissions. You will receive a notification by email when this is completed.
```

Comment:
We should probably setup this so-called notification by email business... Maybe a post save hook on users? If they didn't have groups and now they have groups then use django's email framework (examples are in the workflow callbacks on studies) to send an email which Nici should hopefully have a template for. I'll make a ticket.